### DISCUSSION TITLE
Is it possible to only push a language added on Weblate after it reaches a % completeness?

### DISCUSSION THREAD

**Discussion by papjul** [WeblateOrg/weblate#9719] [Ideas] [answered]
I'm sorry if this question has already been brought, I couldn’t find it in issues or in documentation, I tried with "percent" and "completeness" keywords.

Here is how I would see it:
- I define a % to reach for a language to be valid, for example 30%
- When a user add a new language ***on Weblate***, it will only be pushed to app repo after it reaches at least 30%
- If there are existing languages on app repo that are at 34%, but let’s say it drops to 28% because it becomes unmaintained, then new changes should still be pushed (for example, if someone adds new contributions that make it 29%)

Does something like that exist? Or what would you think of implementing it?

I'm asking it because it’s the third time I have translators adding new languages but they don't translate anything (or just one string). Pushing that new language to app repo make the false believe to users of the app that the translation exists and when they will load it, they will just see the English translation as fallback.

Thank you!

**Comment by nijel** [comment, marked as answer]
No such feature currently exists. There is https://github.com/WeblateOrg/weblate/issues/3436 covering this topic.

**Reply by papjul** [reply]
Thank you, indeed it seems to be an Android-specific issue as Android picks up any translation he finds.
